README.md

```diff
@@ -19,24 +19,22 @@ allprojects {
 
 After that add library to your module:
 ```groovy
-implementation 'com.redmadrobot.gallery:android-gallery:latest.version.here'
+  implementation 'com.github.oguzhanaslann:android-gallery:<latest.version.here>'

```

Comment:
So, judging by this pull request, I assume that you've made minor refactoring in our library and trying to replace package name in readme from ours package name into yours. 🤔